### PR TITLE
Unforking XStream

### PR DESCRIPTION
**Pull request by jglick**
Background for https://github.com/jenkinsci/jenkins/pull/4944. FYI @joehni

### CODE REVIEW

File: jep/0000/README.adoc
Comment (oleg-nenashev) [inline jep/0000/README.adoc:106]: It may worth double-checking Custom Tools Plugin. I do not recall logic there, but has quite complex tool initialization logic

**Review by oleg-nenashev — APPROVED** [review]
Looks good from the JEP draft requirements.

File: jep/0000/README.adoc
Comment (jglick) [reply jep/0000/README.adoc:106]: Handled by https://github.com/jenkinsci/custom-tools-plugin/pull/51 I believe.

**Comment by jglick** [comment]
@jenkinsci/jep-editors ping, it is hard to even discuss this when I do not have a number assigned.

**Review by Wadeck — APPROVED** [review]
👍 Really interesting JEP descriptions, learned a lot of technical details about XStream, thanks!

**Review by MarkEWaite — APPROVED** [review]
Looks ready for the next step.

**Comment by jglick** [comment]
Yes I think so.

File: jep/0000/README.adoc
Comment (oleg-nenashev) [inline jep/0000/README.adoc:76]:
```suggestion
If https://github.com/jenkinsci/jep/blob/master/jep/7/README.adoc[JEP-7] is implemented, then `XStream2.setMapper` and `.getMapperInjectionPoint` can be removed as `ruby-runtime` was the only caller.
```

File: jep/0000/README.adoc
Comment (oleg-nenashev) [inline jep/0000/README.adoc:202]: Needs a community sign-off before merging.
Not a subject for review in the case of JEP Draft acceptance process

File: jep/0000/README.adoc
Comment (oleg-nenashev) [inline jep/0000/README.adoc:70]: All specification went to compatibility notes. Not a blocker, but it looks a bit unusual

**Review by oleg-nenashev — APPROVED** [review]
I am approving this JEP as a copy editor. The JEP-1 requirements are followed though some content is definitely a subject for discussion in the mailing list. Maybe it makes sense to also introduce the rollout plan section.

All comments and suggestions are to be merged in a follow-up. I will start the PR integration process right away

File: jep/0000/README.adoc
Comment (jglick) [reply jep/0000/README.adoc:202]: Sign off on what exactly?

File: jep/0000/README.adoc
Comment (jglick) [reply jep/0000/README.adoc:70]: Not sure what you mean?

**Pull request merged**
